### ISSUE TITLE
UX and WCAG issues with login form

### ISSUE THREAD

**Issue by NiklasBr** [pimcore/studio-ui-bundle#579]
![image](https://github.com/user-attachments/assets/47d5acd7-3d5c-4cf0-a719-e368460ec31e)

A few notes on how it can be improved:

 - Add descriptive `<label>`s
 - Regression from Classic: The `autocomplete` [attributes are missing](https://github.com/pimcore/admin-ui-classic-bundle/blob/c68397e4e4fa83d6179e1a04b75fc28f2747a4ac/templates/admin/login/login.html.twig#L18-L20), making it less compatible with password managers and the like.
 - Once you *do* autofill, it is clear that it is the wrapper element which is styled as an input element and not the actual input, making it look ugly. See the light blue background that e.g. 1Password adds.

**Comment by markus-moser** [comment]
> * Add descriptive `<label>`s

We added aria labels now.

> * Regression from Classic: The `autocomplete` [attributes are missing](https://github.com/pimcore/admin-ui-classic-bundle/blob/c68397e4e4fa83d6179e1a04b75fc28f2747a4ac/templates/admin/login/login.html.twig#L18-L20), making it less compatible with password managers and the like.
  
Autocomplete attributes are now added.
  
> * Once you _do_ autofill, it is clear that it is the wrapper element which is styled as an input element and not the actual input, making it look ugly. See the light blue background that e.g. 1Password adds.

Unfortunately we cannot solve this because of the way how ant design works. We need to keep this as it is (for now).
